src/pages/index.astro

```diff
@@ -0,0 +1,12 @@
+---
+import Layout from "../layouts/Layout.astro";
+import Hero from "../components/Hero.astro";
+import Info from "../components/Info.astro";
+import OpenSourceBlurb from "../components/OpenSourceBlurb.astro";
+---
+
+<Layout title="Grain Programming Language">
+	<Hero />
+	<Info />
```

Comment:
Why was the `build with us` section from the figma excluded?

Reply:
It just seemed like redundant information given the rest of what's on the homepage